pkg/sidecar/k8s_instance.go

```diff
@@ -322,6 +323,7 @@ func (n *K8sNetwork) ConfigureNetwork(ctx context.Context, cfg *sync.NetworkConf
 		if err != nil {
 			return fmt.Errorf("failed to generate new network config list: %w", err)
 		}
+		logging.S().Debugw("Jim", "netconf", netconf)
```

Comment:
@jimpick heads up, that at larger clusters we seem to be having issues with this part and it is not yet clear why: https://github.com/testground/testground/pull/924

Looks like currently it is possible to block Weave somehow.